util/uvmdvgen/scoreboard.sv.tpl

```diff
@@ -69,7 +69,7 @@ class ${name}_scoreboard extends dv_base_scoreboard #(
       csr = ral.default_map.get_reg_by_offset(csr_addr);
       `DV_CHECK_NE_FATAL(csr, null)
     end
-    end else begin
+    else begin
```

Comment:
In the RTL, we usually have the `end else begin` on a single line. Did you use some formatting tool here?

Reply:
Not formatting tool - just fixing a bug due to `end` duplicated on line 71 and 72. 

I would guess this comes from SW coding style that mandates `} else {` on the same line. Since SV is more 'wordy' in this regard, I've personally not been a fan of `end else begin` on the same line, considering DV code can be quite complex when looked at. Starting the `else` on a new line helps keep conditional branch blocks separate visually. Plus, if we label the begin & end, we'd have to start the else on a separate line, resulting in a mix of both styles in our code base. My preference is to relax this requirement, unless people are strongly opposed to it.